app/templates/show-project.hbs

```diff
@@ -119,90 +119,95 @@
           <p class="text-small">
-            <strong>FEMA Flood Zone:</strong>
-            {{#if model.dcp_femafloodzonea}}<span class="label secondary">Zone A</span>{{/if}}
-            {{#if model.dcp_femafloodzonecoastala}}<span class="label secondary">Coastal Zone A</span>{{/if}}
-            {{#if model.dcp_femafloodzoneshadedx}}<span class="label secondary">Zone Shaded X</span>{{/if}}
-            {{#if model.dcp_femafloodzonev}}<span class="label secondary">Zone V</span>{{/if}}
+            <strong>Addresses:</strong>
+            {{#each model.addresses as |address index|}}
+              {{if index ", "}}{{address.dcp_validatedaddressnumber}} {{address.dcp_validatedstreet}}
+            {{/each}}
           </p>
-        {{/if}}
 
-        {{#if model.dcp_bsanumber}}
           <p class="text-small">
-            <strong>BSA Number<sup class="dark-gray">{{icon-tooltip tip='Board of Standards and Appeals Calendar Number'}}</sup>:</strong>
-            {{model.dcp_bsanumber}}
+            <strong>Borough:</strong>
+            {{model.dcp_borough}}
+            {{#if model.dcp_sisubdivision}}<small><span class="label light-gray">Subdivision</span></small>{{/if}}
+            {{#if model.dcp_sischoolseat}}<small><span class="label light-gray">School Seat</span></small>{{/if}}
           </p>
-        {{/if}}
 
-        {{#if model.dcp_wrpnumber}}
           <p class="text-small">
-            <strong>WAP Number<sup class="dark-gray">{{icon-tooltip tip='Waterfront Revitalization Program Application Number'}}</sup>:</strong>
-            {{model.dcp_wrpnumber}}
+            <strong>Lead Division:</strong>
+            {{lookup-dcp-division model.dcp_leaddivision}}
           </p>
-        {{/if}}
 
-        {{#if model.dcp_lpcnumber}}
           <p class="text-small">
-            <strong>LPC Number<sup class="dark-gray">{{icon-tooltip tip='Landmark Preservation Commission Number'}}</sup>:</strong>
-            {{model.dcp_lpcnumber}}
+            <strong>Keywords:</strong>
+            {{#if model.keywords}}
+              {{#each model.keywords as |keyword|}}
+                <span class="label secondary">{{keyword}}</span>
+              {{/each}}
+            {{else}}
+              None
+            {{/if}}
           </p>
-        {{/if}}
 
-        {{#if model.dcp_nydospermitnumber}}
+          {{#if (or model.dcp_ceqrtype model.dcp_ceqrnumber)}}
+            <p class="text-small">
+              <strong>CEQR<sup class="dark-gray">{{icon-tooltip tip='City Environmental Quality Review'}}</sup>:</strong>
+              {{#if model.dcp_ceqrtype}}<span class="label secondary">{{model.dcp_ceqrtype}}</span>{{/if}}
+              {{#if model.dcp_ceqrnumber}}<span class="label secondary">{{model.dcp_ceqrnumber}}</span>{{/if}}
+            </p>
+          {{/if}}
+
+          {{#if (or model.dcp_femafloodzonea model.dcp_femafloodzonecoastala model.dcp_femafloodzoneshadedx model.dcp_femafloodzonev) }}
```

Comment:
For conditional stuff (specifically handling for missing or incomplete data), we should probably try to create a computed with in the model (`model.isFloodZone`))